FormType/RecipientsType.php

```diff
@@ -50,13 +51,13 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
      */
     public function getParent()
     {
-        return 'text';
+        return TextType::class;
```

Comment:
``LegacyFormHelper::isLegacy() ? 'text' : 'Symfony\Component\Form\Extension\Core\Type\TextType';`` we can't use short notation yet.